src/patternfly/components/Hint/hint.scss

```diff
@@ -0,0 +1,114 @@
+.pf-c-hint {
+  --pf-c-hint__header--GridColumnWidth: auto;
+  --pf-c-hint__actions--GridColumnWidth: calc(var(--pf-global--spacer--2xl) + #{pf-size-prem(44px)});
```

Comment:
where does this value come from? Why not just let it be the size of the action + the purple margin/gap?

Reply:
From this design, looks like this column width should fit whatever's placed in it

<img width="632" alt="Screen Shot 2020-06-25 at 5 09 54 PM" src="https://user-images.githubusercontent.com/35148959/85800759-29abcb00-b707-11ea-9052-e9b3fec400d7.png">

Reply:
updated